course_project/23/src/main/java/com/example/course_project/UsersRepository.java

```diff
@@ -0,0 +1,32 @@
+package com.example.course_project;
+
+import org.springframework.stereotype.Repository;
+
+import java.sql.SQLException;
+import java.util.Optional;
+
+@Repository
+public class UsersRepository extends ConnectionRepository {
```

Comment:
This will do now with the connection and prepareStatement, but I advise in the future to read up on named parameter jdbc template (will save you some time)